docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
     stdin_open: true
     volumes:
       - ./front-end:/usr/src/app/frontend:cached
+      - /usr/src/app/frontend/node_modules
```

Comment:
Saw issue when new dependencies were being adding, it seemed to be syncing node modules from local machine instead of container